src/material-examples/table-expandable-rows/table-expandable-rows-example.html

```diff
@@ -1,7 +1,7 @@
 <table mat-table
        [dataSource]="dataSource" multiTemplateDataRows
        class="mat-elevation-z8">
-  <ng-container matColumnDef="{{column}}" *ngFor="let column of columnsToDisplay">
+  <ng-container matColumnDef="column" *ngFor="let column of columnsToDisplay">
```

Comment:
These changes end up throwing an error, because the `column` name is dynamic. The correct way to solve it would be to change it to `[matColumnDef]="column"`.